packages/blueprints-integration/src/migrations.ts

```diff
@@ -129,6 +151,11 @@ export interface MigrationStepCore extends MigrationStep {
 	migrate?: MigrateFunctionCore
```

Comment:
Note for myself: What are these `*Core` ones used for if they arent for the system blueprint?